Sail/src/Sail/entities/EntityFactory.cpp

```diff
@@ -64,7 +64,7 @@ void EntityFactory::CreateCandle(Entity::SPtr& candle, const glm::vec3& lightPos
 	PointLight pl;
 	pl.setColor(glm::vec3(0.55f, 0.5f, 0.45f));
 	pl.setPosition(glm::vec3(lightPos.x, lightPos.y + .5f, lightPos.z));
-	pl.setAttenuation(0.f, 0.f, 0.2f);
+	pl.setAttenuation(10.f, 0.f, 0.2f);
```

Comment:
Is this correct or is this change due to some testing?

Reply:
Yes it is correct. Altough the first attenuation value is not used as attenuation anymore, it specifies the light radius. I will make some changes to make this more clear!